src/buckets/components/context/lineProtocol.tsx

```diff
@@ -68,6 +68,20 @@ export const LineProtocolProvider: FC<Props> = React.memo(({children}) => {
     setWriteError('')
   }
 
+  /**
```

Comment:
This file updated based on https://github.com/influxdata/ui/pull/2919 to clear up a typescript check error since we are using the most recent version of the `openapi` spec in `OSS-2.1`. All the other files are cherry-picked from https://github.com/influxdata/ui/pull/3039